src/main/java/com/petfinder/controller/AdvertisementController.java

```diff
@@ -0,0 +1,79 @@
+package com.petfinder.controller;
+
+import com.petfinder.service.AdvertisementService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class AdvertisementController {
+
+    @Autowired
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/matikz1110/PetFinder/file/2528026134/issues/source?bid=3239464&fileBranchId=3282657#l16)